src/objects.js

```diff
@@ -1,41 +1,78 @@
 const createPerson = (name, age) => {
-  // your code here
+  const newPerson = new Object();
+  newPerson.name = name;
+  newPerson.age = age;
+
+  return newPerson;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object.hasOwnProperty(property)) {
+    return true;
+  }
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(persons => persons.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  for (let i = 0; i < people.length; i++) {
+    const person = people[i];
+    if (person.name === name) {
+      return person;
+    }
+  }
+  return null;
```

Comment:
You could use the built in filter method here for this:
```
  return people.filter(person => person.name === name)[0];
```
This gets around the eslint error coming up because there is no return value at the end of the function